samples/client_publish_acl.py

```diff
@@ -27,7 +27,7 @@ async def test_coro() -> None:
         logger.info("messages published")
         await C.disconnect()
     except ConnectError as ce:
-        logger.exception(f"Connection failed: {ce}")
+        logger.exception("Connection failed")
```

Comment:
```suggestion
        logger.exception("Connection failed", exc_info=True)
```

Reply:
`logger.exception` is actually an alias of `logger.error(..., exc_info=True)`